modules/images-configuration-parameters.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration.adoc
+
+[id="images-configuration-parameters_{context}"]
+= Image controller configuration parameters
+
+The `image.config.openshift.io/cluster` resource offers the following
+configuration parameters.
+
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`metadata`
+|Standard object's metadata.
+
+|`spec`
+|Holds user-settable values for configuration.
+
+|`status`
+|Holds observed values from the cluster.
+
+|`allowedRegistriesForImport`
+|Limits the container image registries from which normal users may import images.
+Set this list to the registries that you trust to contain valid Docker images,
+and that you want applications to be able to import from. Users with permission
+to create images or `ImageStreamMappings` from the API are not affected by this
+policy. Typically only cluster administrators will have the appropriate permissions.
+
+|`externalRegistryHostnames`
+|Provides the hostnames for the default external image registry. The external
+hostname should be set only when the image registry is exposed externally. The
+first value is used in 'publicDockerImageRepository' field in ImageStreams.
+The value must be in `hostname[:port]` format.
+
+|`additionalTrustedCA`
+|A reference to a ConfigMap containing additional CAs that should be trusted
+during ImageStream import, pod image pull, and `openshift-image-registry`
```

Comment:
Should these be more along the lines of: 

`ImageStream import`, `pod image pull`, and `openshift-image-registry pullthrough`

Reply:
i guess